specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/examples/Endpoints_Create.json

```diff
@@ -40,7 +40,7 @@
                       "192.168.1.0/24",
                       "10.0.0.0/24"
                     ],
-                    "@odata.type": "#Microsoft.Azure.Cdn.Models.DeliveryRuleRemoteAddressConditionParameters"
+                    "typename": "DeliveryRuleRemoteAddressConditionParameters"
```

Comment:
Per the email discussion, we agreed to utilize camel casing for type name. Please utilize typeName instead and change for subsequent fields that have not yet been changed.

Reply:
Updated